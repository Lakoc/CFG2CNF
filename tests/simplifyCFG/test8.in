S,A,B
a,b,c
S
S->B
B->A
B->cBB
A->bA
A->S
