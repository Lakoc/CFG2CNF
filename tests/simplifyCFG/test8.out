S,A,B
a,b,c
S
S->bA
S->cBB
B->bA
B->cBB
A->bA
A->cBB
